### PR TITLE
Docker environment

### PR DESCRIPTION
**Pull request by EvanAntoni**
Added configuration for deployment on Docker. Also added add-on for docker compose 
for simplified management of application services.

### CODE REVIEW

**Review by RazSKg — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM python:2.7.15
```
Comment (serge30) [inline Dockerfile]: `python:2.7` is enough.

File: frontend/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM node:10.15.0
```
Comment (serge30) [inline frontend/Dockerfile]: `node:10.15` is enough. Please try do not set hotfix version and use only parts. As there might be new hotfix releases which fixes some vulnerabilities and bugs, and you'd like to pick them.

**Review by serge30 — CHANGES_REQUESTED** [review]
Please update `README.md` with steps to deploy the project on dev environments using your script and Docker.

**Review by serge30 — APPROVED** [review]

**Comment by EvanAntoni** [comment]
- Fixed version of images
- Added deployment instructions through Docker into README.md

Additionally:
- Corrected help in the deployment script
- Made minor changes to the docker-compose.yml and development.ini

**Pull request merged**